package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "vets-json-schema",
-  "version": "20.19.2",
+  "version": "20.19.3",
```

Comment:
I'm not sure if the version needs to be update or not. There aren't any real changes to the schemas besides updating the json files in dist